docs/ACM_SWITCHOVER_RUNBOOK.md

```
@@ -321,6 +329,46 @@ done
 
 ---
 
+### Step 4b: Set Auto-Import Strategy to ImportAndSync (ACM 2.14+ with Existing Clusters)
+
+> **Perform this step on your destination hub if it has any non local-cluster managed clusters and if you plan to switch back to the primary hub in the future.**
+
```

markdownlint-cli2 (0.18.1)

Blank line inside blockquote

(MD028, no-blanks-blockquote)